modules/consul-cluster/main.tf

```diff
@@ -313,7 +313,7 @@ resource "aws_security_group" "lc_security_group" {
     from_port       = "22"
     to_port         = "22"
     protocol        = "tcp"
-    security_groups = ["${var.ssh_security_group_id}"]
+    security_groups = ["${var.ssh_security_group_ids}"]
```

Comment:
@danvaida I think `[` `]` are obsoletes as `ssh_security_group_ids` has  type `list`

Reply:
Don't forget that this is TF `0.11` :)

Just got to the examples dir and make this change and see how the plan returns:
```shell
Error: module.aws_vault.module.consul_cluster.aws_security_group.lc_security_group: ingress.16.security_groups: should be a list
Error: module.aws_vault.module.vault_cluster.aws_security_group.lc_security_group: ingress.7.security_groups: should be a list
```